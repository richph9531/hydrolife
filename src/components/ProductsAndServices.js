import React from 'react';

import classes from '../styles/ProductsAndServices.module.css';

const ProductsAndServices = () => (
  <div className={classes.ProductsAndServices}>
  <h1>Products and Services</h1>
</div>
);

export default ProductsAndServices;
