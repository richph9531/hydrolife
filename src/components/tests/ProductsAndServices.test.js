import React from 'react';

import { render } from '@testing-library/react';
import ProductsAndServices from '../ProductsAndServices';

test('Products and Services renders correctly', () => {
  const view = render(<ProductsAndServices />);
  expect(view.baseElement).toMatchSnapshot();
});
