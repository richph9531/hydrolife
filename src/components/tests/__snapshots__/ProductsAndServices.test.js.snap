// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products and Services renders correctly 1`] = `
<body>
  <div>
    <div
      class="ProductsAndServices"
    >
      <h1>
        Products and Services
      </h1>
    </div>
  </div>
</body>
`;
